### DISCUSSION TITLE
Definition of Platelet  in Peripheral Blood Mononuclear Cells (PBMC)

### DISCUSSION THREAD

**Discussion by gengjie2000** [satijalab/seurat#8693] [General]
Hello everyone, I am a beginner in bioinformatics. Recently, I have learned a lot from the single-cell analysis process (https://satijalab.org/seurat/articles/pbmc3k_tutorial). However, I have a question and would like to ask everyone. Can a cluster of cells be defined as platelets in Peripheral Blood Mononuclear Cells (PBMC)? Because platelets and megakaryocytes are often absent in the circulatory system. And there is no RNA in anucleated platelets. This makes me feel very confused. In fact, in my data, I have also clustered a group of cells that express the platelet marker (PPBP). I hope everyone can help me solve this confusion. Welcome to discuss, thank you.
![image](https://github.com/satijalab/seurat/assets/130119573/052196c5-31b3-49d1-802b-53984d64da6d)

**Comment by samuel-marsh** [comment]
Hi,

I'm going to move this to the discussion section as Issues section is mainly for bugs or issues running the R package and not biological questions.

Best,
Sam
